migrations/master/README.md

```diff
@@ -61,3 +61,78 @@ services:
 ```
 
 You might need to reuse or adapt those changes for your setup.
+
+
+## cas
+From 22.x to 23.x
+To activate password rotation policy for ldap, user need to run these commands:
+
+```
+ldapmodify -H ldap://  -D cn=admin,dc=georchestra,dc=org -w secret
+dn: cn=module{0},cn=config
+changetype: modify
+add: olcModuleLoad
+olcModuleLoad: ppolicy.la
```

Comment:
fwiw in 23.0 ppolicy module is already loaded since https://github.com/georchestra/georchestra/blob/31d5a963d5d949b30e36295f133ccc3af8c86aa2/ldap/docker-root/lastbind.ldif#L6 so im not sure this bit is needed

you have some hardcoded `172.23.0.11`  below :)

Reply:
yep, will update code and check if necessary